packages/lesswrong/server/resolvers/commentResolvers.ts

```diff
@@ -192,4 +200,82 @@ augmentFieldsDict(Comments, {
       return (post && post.contents && post.contents.version) || "1.0.0";
     },
   },
+  doppelComments: {
+    resolveAs: {
+      type: '[DoppelComment]',
+      resolver: async (dbComment, context: ResolverContext) => {
+        const user = await Users.findOne({ _id: dbComment.userId });
+        if (!user) return []
+        if (user.karma < 1e4) return []
+        const doppelComments = await runFragmentQuery({
+          fragmentName: 'DoppelCommentsFragment',
+          terms: { view: 'doppelCommentsForComment', commentId: dbComment._id },
+          collectionName: 'DoppelComments',
+        })
+
+
+        if (doppelComments.length < 2) {
+          // We'll make 'em for next time
+          void [createDoppelComment(dbComment, user), createDoppelComment(dbComment, user)]
+        }
+        return doppelComments
```

Comment:
let's come up with some other way to populate them

Reply:
(idk what the exact intent here is, or when they normally get populated; we can discuss options)